components/homepage/pricing.tsx

```diff
@@ -0,0 +1,209 @@
+"use client";
```

Comment:
I think I have this page elsewhere in my app. So I think we can remove it. Not certain though so we should double check.